bioimageio/core/prediction.py

```diff
@@ -270,8 +279,10 @@ def check_tiling(tiling):
         spatial_axes = [ax for ax in axes if ax in "xyz"]
         halo = tiling["halo"]
         tile = tiling["tile"]
+        scale = tiling.get("scale", dict())
         assert all(halo.get(ax, 0) >= 0 for ax in spatial_axes)
         assert all(tile.get(ax, 0) > 0 for ax in spatial_axes)
+        assert all(scale.get(ax, 1.0) > 0 for ax in spatial_axes)
```

Comment:
Can you get `output.scale` here as default instead of `1.0`? Then this would also yield correct results for cases where the scale changes and where users specify custom parameters for halo and tile (but not for scale).

Reply:
hmm, I changed this a bit more. It seemed awkward to have `scale.get(ax, output_spec.shape.scale.get(ax, 1)) > 0`. Definitely doable if that's the right approach. Instead I ensured that any metadata provided in output_spec is read into the tiling dict first, and then overwritten with whatever the users provides, which should achieve the same thing. Let me know if you think that more consequences than just fixing this issue as now a user could provide a custom "halo", "tile", or "scale" term and in all cases the anything not provided will fall back to the defaults that would have been generated on a `tiling=True`

Reply:
Yes, I think that's much better.